cockatrice/src/tab_game.cpp

```diff
@@ -802,6 +802,27 @@ Player *TabGame::addPlayer(int playerId, const ServerInfo_User &info)
     gameMenu->insertMenu(playersSeparator, newPlayer->getPlayerMenu());
 
     players.insert(playerId, newPlayer);
+
+    int i = 1;
+    // Loop for each player, the idea is to have one assigned zone for each non-spectator player
+    for (i = 1; i <= players.count(); ++i) {
+        bool aPlayerHasThisZone = false;
+        QMapIterator<int, Player *> playerIterator(players);
+        while (playerIterator.hasNext()) {
```

Comment:
we prefer `for (auto &item : array)` loops over java style iterators in new pull requests